### DISCUSSION TITLE
How to set the winuae settings.

### DISCUSSION THREAD

**Discussion by pi1541** [BartmanAbyss/vscode-amiga-debug#178] [Q&A] [answered]
Pardon my ignorance, but how do I get it to launch winuae with a configuration that has some actual fast mem?
I understand how to change the configuration between A500, A1200 and A4000 but none of them have any fast mem.
Thanks.

**Comment by BartmanAbyss** [comment, marked as answer]
Hi, have you had a look at this section? https://github.com/BartmanAbyss/vscode-amiga-debug#supported-amiga-models

**Reply by pi1541** [reply]
Thanks!
